src/kixi/heimdall/invites.clj

```diff
@@ -1,18 +1,51 @@
 (ns kixi.heimdall.invites
-  (:require [taoensso.timbre :as log]
+  (:require [clojure.spec.alpha :as s]
+            [clojure.spec.gen.alpha :as gen]
```

Comment:
I do wonder about having a `kixi.heimdall.invites.spec` namespace, tho there might be some circular references.